ui/apps/platform/src/Containers/VulnMgmt/RiskAcceptance/useImageVulnerabilities.ts

```diff
@@ -3,30 +3,36 @@ import {
     GetImageVulnerabilitiesData,
     GetImageVulnerabilitiesVars,
     GET_IMAGE_VULNERABILITIES,
+    GET_IMAGE_VULNERABILITIES_LEGACY,
 } from './imageVulnerabilities.graphql';
 
-function useImageVulnerabilities({ imageId, vulnsQuery, pagination }) {
+function useImageVulnerabilities({ imageId, vulnsQuery, pagination, showVMUpdates = false }) {
```

Comment:
I don't think you need to pass the `showVMUpdates` value if you use the feature flag hook within this hook